corehq/apps/domain/dbaccessors.py

```diff
@@ -3,6 +3,17 @@
 from corehq.util.test_utils import unit_testing_only
 
 
+def get_doc_count_in_domain_by_class(domain, doc_class):
+    doc_type = doc_class.__name__
+    row = doc_class.get_db().view(
+        "by_domain_doc_type_date/view",
+        startkey=[domain, doc_type],
+        endkey=[domain, doc_type, {}],
+        reduce=True,
```

Comment:
Thanks, this looks great! I'd also look for where `get_doc_ids_in_domain_by_class` is tested and add another similar test right beside it for this one. (I'm so happy every time I refactor a dbaccessor because they're all tested!)